Source/WebCore/Modules/mediastream/gstreamer/GStreamerPeerConnectionBackend.cpp

```diff
@@ -80,7 +80,7 @@ static RTCRtpCapabilities gstreamerRtpCapatiblities(const String& kind)
             .sdpFmtpLine = emptyString() });
     } else {
         capabilities.headerExtensions.append({ "urn:3gpp:video-orientation"_s });
-        capabilities.codecs.reserveCapacity(4);
+        capabilities.codecs.reserveCapacity(6);
```

Comment:
Isn't 5 enough?

Reply:
No, in main it's wrong already, we reserve for 4 but append 5.